$MirOS: ports/multimedia/mplayer/patches/patch-libmpdemux_demux_asf_c,v 1.1.7.1 2005/03/18 15:50:49 tg Exp $
--- libmpdemux/demux_asf.c.orig	Sat Jan 15 16:11:22 2005
+++ libmpdemux/demux_asf.c	Thu May 19 20:32:06 2005
@@ -350,7 +350,8 @@ int demux_asf_fill_buffer(demuxer_t *dem
             return 1; // success
     }
     
-    mp_msg(MSGT_DEMUX,MSGL_V,"%08X:  UNKNOWN TYPE  %02X %02X %02X %02X %02X...\n",demux->filepos,asf_packet[0],asf_packet[1],asf_packet[2],asf_packet[3],asf_packet[4]);
+    mp_msg(MSGT_DEMUX,MSGL_V,"%08X:  UNKNOWN TYPE  %02X %02X %02X %02X %02X...\n",
+      (int)demux->filepos,asf_packet[0],asf_packet[1],asf_packet[2],asf_packet[3],asf_packet[4]);
     return 0;
 }
 
